styles/styles.css

```diff
@@ -0,0 +1,37 @@
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  width: 310px;
```

Comment:
Do you want to change the width for any text changes? Is this size important here?